src/process/process.rs

```diff
@@ -15,15 +15,15 @@ extern {
 }
 
 /// Size of each processes's memory region in bytes
-pub const PROC_MEMORY_SIZE : usize = 8192;
-pub const NUM_PROCS : usize = 2;
+pub const PROC_MEMORY_SIZE : usize = 2048;
+pub const NUM_PROCS : usize = 1;
```

Comment:
Do two not fit right now? We gotta find a way to make this platform specific (shouldn't be to hard with a bit of refactoring), but for now, merging this as is would mean the Firestorm wouldn't run multiple apps.

For now, if we can fit 2 instead of 1, I think that will be fine until we make a more permanent change (the Firestorm apps should be able to live with 2K no problem for now).

One option is to make all of these extern variables from the platform crate instead of constants.